apps/browser/src/_locales/en/messages.json

```
@@ -1071,6 +1071,10 @@
     },
     "description": "Aria label for the view button in notification bar confirmation message"
   },
+  "notificationEditToolTip": {
```

super nitty, non-blocking nit:

```suggestion
  "notificationEditTooltip": {
```